app/services/get_application_choices_ready_to_send_to_provider.rb

```diff
@@ -0,0 +1,21 @@
+class GetApplicationChoicesReadyToSendToProvider
+  def self.call
+    application_choices_past_edit_by(
```

Comment:
Personal preference, I like it more procedural:

```rb
scope = ApplicationChoice.where(status: :application_complete)
scope = application_choices_past_edit_by(scope)
scope = application_choices_with_references_complete(scope)
```

WDYT?

Reply:
Yes I think that's better, if it were Elixir they have a nice syntax for this :)